lib/apnotic/request.rb

```diff
@@ -16,7 +16,11 @@ def build_headers_for(notification)
       h.merge!('apns-id' => notification.apns_id) if notification.apns_id
       h.merge!('apns-expiration' => notification.expiration) if notification.expiration
       h.merge!('apns-priority' => notification.priority) if notification.priority
-      h.merge!('apns-push-type' => notification.background_notification? ? 'background' : 'alert' )
+      if notification.type
+        h.merge!('apns-push-type' => notification.type)
+      else
+        h.merge!('apns-push-type' => notification.background_notification? ? 'background' : 'alert' )
```

Comment:
I would defer to the owner here but here we have two ways of doing the `if` statement for the same function... I would suggest...

```
if notification.type
  ..
elsif notification.background_notification?
  ...
else
  ...
end
```

Reply:
Yes, I was trying to maintain some backwards compatibility in cases where the type is not set, using the approach you're proposing allows us to be more specific and would allow us to just set `voip` type if we want to eliminate generic `type`